tembo-py/README.md

```diff
@@ -2,11 +2,129 @@
 
 The official Python client for Tembo.io
 
+## Table of Contents
+
+- [Prerequisites](#prerequisites)
+- [Installation](#installation)
+- [Prepare Contextual Basis](#prepare-contextual-basis)
+- [Adding Custom Prompts](#adding-custom-prompts)
+
+## Prerequisites
+
+- [Python](https://www.python.org/) - Programming language and companion `pip` package manager
 
 ## Installation
 
+The [tembo-py library](https://pypi.org/project/tembo-py/) is hosted on pypi.org and can be installed using the following `pip` command.
+
 ```bash
 pip install tembo-py
 ```
 
+## Prepare Contextual Basis
+
+Before jumping in, it's important to have material to offer the model as context.
+The [RAG Stack official documentation](https://tembo.io/docs/tembo-stacks/rag#build-a-support-agent-with-tembo-rag) does a good job reviewing this in detail, so keep the following points brief.
+
+```python
+from tembo_py.rag import TemboRAG
+
+rag = TemboRAG(
+    project_name="tembo_support",
+    chat_model="gpt-3.5-turbo",
+    connection_string="postgresql://postgres:<your-password>@<your-TemboHost>:5432/postgres"
+)
+
+chunks = rag.prepare_from_directory("./tembo_docs") # File path to your loadable data
+
+rag.load_documents(chunks)
+```
+
+Now that the table is loaded into Postgres, you can run the following:
+
+```python
+rag.init_rag(connection_string="postgresql://postgres:<your-password>@<your-TemboHost>:5432/postgres",
```

Comment:
`connection_string` is not needed here if it is provided during initialization of `TemboRAG`
```suggestion
rag.init_rag(
```